modules/RTC/TPCUtils.js

```diff
@@ -445,6 +445,19 @@ export class TPCUtils {
                     this.pc.localSSRCs.delete(oldTrack.rtcId);
                 });
         } else if (newTrack && !oldTrack) {
+            if (newTrack.getVideoType() === VideoType.DESKTOP) {
+                const transceiver = this.pc.peerconnection
```

Comment:
This picks the 2nd transceiver and puts the screen track on it. For the purpose of POC where camera is already in the connection when screensharing is started.